src/microbe_stage/OrganelleDefinition.cs

```diff
@@ -271,14 +271,23 @@ public enum OrganelleGroup
     /// <summary>
     ///   Gets the visual scene that should be used to represent this organelle (if there is one)
     /// </summary>
-    /// <para cref="modelInfo">
+    /// <param name="upgrades">
+    ///   Some upgrades alter organelle visuals
+    /// </param>
```

Comment:
```suggestion
    /// <param name="upgrades">Some upgrades alter organelle visuals</param>
```
This fits on a single line.